test/tools/ossfuzz/strictasm_diff_ossfuzz.cpp

```diff
@@ -97,6 +97,9 @@ extern "C" int LLVMFuzzerTestOneInput(uint8_t const* _data, size_t _size)
 		(yul::test::yul_fuzzer::yulFuzzerUtil::maxSteps * 4)
 	);
 
+	if (yulFuzzerUtil::resourceLimitsExceeded(termReason))
+		return 0;
+
```

Comment:
Why use `termReason` instead of letting the exceptions escape? That would ensure that this cannot be missed.

Reply:
Three of the four exceptions need to be handled one way (bail out early), and the other needs to be handled another way (not bail out but continue). Something like

```
        catch (StepLimitReached const&)
        {
             return 0;
        }
        catch (TraceLimitReached const&)
        {
             return 0;
        }
        catch (ExpressionNestingLimitReached const&)
        {
             return 0;
        }
        catch (ExplicitlyTerminated const&)
        {
        }
```

And similarly with the other harness. Perhaps this way we avoid code duplication i.e., not repeating catch clauses.

Reply:
Also, each harness will then have 8 catch clauses which makes it somewhat verbose?